doc/source/tune/tutorials/tune-resources.rst

```diff
@@ -88,7 +113,13 @@ See :ref:`start-ray-cli` for more information about ``ray.init``:
 
     # Connect to an existing distributed Ray cluster
     ray.init(address=<ray_address>)
-    tuner = tune.Tuner(tune.with_resources(trainable, tune.PlacementGroupFactory([{"CPU": 2, "GPU": 1}])), tune_config=tune.TuneConfig(num_samples=100))
+    # We choose to use a `PlacementGroupFactory` here to specify trial resources
+    resource_group = tune.PlacementGroupFactory([{"CPU": 2, "GPU": 1}])
+    trainable_with_resources = tune.with_resources(trainable, resource_group)
```

Comment:
nit: I know this was already like this, but is there reason why this one doesn't just use a dictionary?